Makefile

```diff
@@ -271,4 +271,24 @@ dockerized-build-amd64:
 
 # Use vendor instead of proxy when building w/ vendor folder
 dockerized-build-vendor:
-	$(DOCKER_BUILD_FROM_SOURCE) --build-arg GO111MODULE=off .
\ No newline at end of file
+	$(DOCKER_BUILD_FROM_SOURCE) --build-arg GO111MODULE=off .
+
+.PHONY: create-eks-cluster
+create-eks-cluster:
+	@eksctl create cluster \
+		--name $(namespace) \
+		--region us-east-1 \
+		--zones us-east-1a,us-east-1b
+
+.PHONY: destroy-eks-cluster
+destroy-eks-cluster:
+	@eksctl delete cluster --region=us-east-1 --name=$(namespace)
+
+.PHONY: deploy-test
+deploy-test:
```

Comment:
eksctl creates a kubeconfig file for you, right? so you could run `make create-eks-cluster` and `make deploy-test` one after another without any extra setup?

Reply:
Ooo that's a good point. Right, maybe you can check the kube context on the machine before running the command.